eng/pipelines/templates/steps/vmr-pull-updates.yml

```diff
@@ -51,7 +51,6 @@ steps:
       --repository "sdk:${{ parameters.targetRef }}"                                                  \
       --recursive                                                                                     \
       --remote "sdk:$(Agent.BuildDirectory)/sdk"                                                      \
-      --component-template $(Agent.BuildDirectory)/sdk/src/VirtualMonoRepo/Component.template.md      \
```

Comment:
delete the file too? or do you want to make it un-templated again?

Reply:
Ah, yeah, let's delete the file too